problem1.ts

```diff
@@ -0,0 +1,26 @@
+// type partFuncOutput = (array: Array<number>) => number | number;
+
+export function sum(args: Array<number>): number {
+  // console.log(args)
+  return args.reduce((akum, next) => akum + next, 0);
+}
+
+export function partFunc(
```

Comment:
I don't think appending 'Func' to the function names is a good practice. While it might work for argument names (e.g., `eachFunc`, `keysFunc`, etc.), I would suggest simplifying the function names by using just `partial`